extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php

```diff
@@ -15,7 +15,15 @@
 							$videoDetails['time'] )->escaped(); ?></div>
 					<div class="video-title"><?= htmlspecialchars( $videoDetails['title'] ); ?></div>
 				</div>
-				<img src="<?= $videoPlayButtonUrl; ?>" class="video-play-button">
+				<div class="video-wiki-play-button">
+					<svg width="22" height="30" viewBox="0 0 22 30"
+					     xmlns="http://www.w3.org/2000/svg">
+						<g fill="none" fill-rule="evenodd">
```

Comment:
If you remove this group, then it should be possible to do what @kvas-damian suggested in https://github.com/Wikia/app/pull/12488/files#r103205998